src/Rings.jl

```diff
@@ -285,8 +285,11 @@ function krull_dim(R::Ring)
   throw(NotImplementedError(:krull_dim, R))
 end
 
+krull_dimension = krull_dim
```

Comment:
I am not sure I like adding this. Does someone really want this? I'd prefer to wait for someone actually requesting this. After all we also have only `dim` and not also `dimension`.

(Ping @thofma @fieker)

That said, if it is decided that we should add these aliases, it needs to be done differently: the above code would lead to any code calling `krull_dimension` to be severely type unstable: after all `krull_dimension` is not `const` and could at any time be replaced by some other value, e.g. a string. So, better:
```suggestion
const krull_dimension = krull_dim
```
Even better: using our `@alias` macro.

Reply:
The suggestion for these aliases was mentioned somewhere in oscar-system/Oscar.jl#4606 and I thought this would be a nice addition - for vector spaces `vector_space_dimension` was already used in some places and so I thought also adding `krull_dimension` would make sense. 
I agree that the current code here could lead to problems as I already encountered them, when defining the alias `vector_space_dimension`. For `vector_space_dimension` I tried to make this better by limiting the scope and I suppose I could also do this here by defining `krull_dimension` only for rings and modules.(I am not how to use the `@alias` macro). 
Alternatively, we could avoid these aliases altogether.

Reply:
I think the offender was `vector_space_dimension`, which should have been `vector_space_dim` from the beginning. (There is also `ambient_dim` etc.)

I don't have a strong opinion on the aliases, but would lean towards not adding them.

Reply:
I have now removed the aliases as they do seem more trouble than they're worth.